### PR TITLE
[haproxy] Fix build failures.

### PR DESCRIPTION
**Pull request by DavidKorczynski**
Fixes broken haproxy build. These commits are particularly relevant for the update:
- https://github.com/haproxy/haproxy/commit/318adf43648143292a6e6b74d4600b4908172598
- https://github.com/haproxy/haproxy/commit/0ff9b3d64fdf0707da8c9a0ae00a7cf753200e36
as they show the relevant code for adapting the fuzzers. 

It is only the `fuzz_hpack_decode` fuzzer that failed.

### CODE REVIEW

**Comment by inferno-chromium** [comment]
Assigning to current sheriff - @oliverchang

File: projects/haproxy/project.yaml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 homepage: "https://github.com/haproxy/haproxy"
-primary_contact: "fuzzing@haproxy.org"
```
Comment (oliverchang) [inline projects/haproxy/project.yaml:2]: Should fuzzing@haproxy.org be moved to auto_ccs instead? Why was it removed completely?

Comment (DavidKorczynski) [reply projects/haproxy/project.yaml:2]: Argh - actually the updates in the `project.yaml` file are not updates at all. This is the state of the `project.yaml` file already (https://github.com/google/oss-fuzz/blob/master/projects/haproxy/project.yaml), so no new changes are in there - apologies!!

Comment (DavidKorczynski) [reply projects/haproxy/project.yaml]: Pardon! Merged with upstream now.

**Comment by inferno-chromium** [comment]
Thanks a lot for these build fixes.

**Pull request merged**